src/tracing/errors.test.ts

```diff
@@ -0,0 +1,70 @@
+import * as Sentry from '@sentry/react'
+
+import { beforeSendAddMechanism, onerror, onunhandledrejection } from './errors'
+
+jest.mock('@sentry/react', () => ({
+  captureException: jest.fn(),
+}))
+
+describe('beforeSendAddMechanism', () => {
```

Comment:
it looks like the coverage isn't being reported for this code. can you add it to the `collectCoverageFrom` set in package.json